src/palace/manager/api/axis.py

```diff
@@ -1967,27 +1995,19 @@ class Axis360AcsFulfillmentInfo(FulfillmentInfo, LoggerMixin):
     and closed the ticket.
     https://github.com/urllib3/urllib3/issues/1677
 
-    This FulfillmentInfo implementation uses the built in Python urllib
+    This Fulfillment implementation uses the built in Python urllib
     implementation instead of requests (and urllib3) to make this request
     to the Axis 360 API, sidestepping the problem, but taking a different
     code path than most of our external HTTP requests.
     """
 
-    def __init__(self, verify: bool, **kwargs: Any) -> None:
-        super().__init__(**kwargs)
-        self.verify: bool = verify
-
-    def problem_detail_document(self, error_details: str) -> ProblemDetail:
-        service_name = urlparse(self.content_link).netloc
-        self.log.warning(error_details)
-        return INTEGRATION_ERROR.detailed(
-            _(RequestNetworkException.detail, service=service_name),
-            title=RequestNetworkException.title,
-            debug_message=error_details,
-        )
+    def __init__(
+        self, content_link: str, content_type: str | None, verify: bool
```

Comment:
Could this `content_type` default to `DeliveryMechanism.ADOBE_DRM`? Would we be calling this with any others?

Reply:
Makes sense. I'll change that.